lib/ajv.d.ts

```diff
@@ -92,7 +92,8 @@ declare namespace ajv {
       data: any,
       dataPath?: string,
       parentData?: Object | Array<any>,
-      parentDataProperty?: string | number
+      parentDataProperty?: string | number,
+      rootData?: any
```

Comment:
Thank you.
No, data can be anything, but parentData and rootData can only be array or object (otherwise you would never get to children).